spec/cypress/integration/Cookies.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Cookie page
+  All ECF users needs to be able to set and modify their cookie preferences
+
+  Scenario: Preferences should not be initially set
+    Given I am on "cookie" page
+    Then "cookie consent" radios are unchecked
```

Comment:
Then "cookie consent" radios are unchecked ->  Then "cookie consent" radio buttons should be unchecked